app/concepcts/cinema_halls/use_cases/update.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module CinemaHalls
+  module UseCases
+    class Update
+      def initialize(id:, params:, repository: CinemaHallRepository.new)
+        @id = id.to_i
+        @repository = repository
+        @params = params
+      end
+
+      def call
+        raise ActiveRecord::RecordNotFound, message if repository.find(id).present? == false
```

Comment:
`message` here is not defined. You have to provide own message or skip it at all :)